.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Is this file meant to be included here?

(I don't use vscode, but we do have tool/editor config files elsewhere in Tock, so not inherently opposed to adding, but if we want to pull in some 'sane editor defaults', probably should be its own PR so people with opinions can take note more easily)

Reply:
No, my mistake. I deleted it and added .vscode to gitignore, may others use vscode.